CoordgenFragmenter.cpp

```diff
@@ -144,6 +144,23 @@ void CoordgenFragmenter::initializeInformation(
     orderFragments(fragments, mainFragment);
     molecule->setMainFragment(mainFragment);
     molecule->setFragments(fragments);
+    for_each(fragments.begin(), fragments.end(), setFlipInfo);
+}
+
+bool CoordgenFragmenter::setFlipInfo(sketcherMinimizerFragment* fragment)
+{
+    fragment->constrainedFlip = false;
+    if (fragment->getAtoms().size() < 2 && fragment->countConstrainedAtoms() == 1) {
+        // fragment has one or two atoms. should only be constrained if a child is constrained
```

Comment:
I'm confused by the comment that mentions fragments with two atoms while the check is getAtoms().size()<2

Reply:
oh yeah you're totally right :) I intended this to be `<=` originally, but I don't think it is necessary. I just updated that comment.  

This does lead me to a question I have about how the flip DOFs work. If we have this mol and scaffold:
<p float="left">
<img width="350" alt="og_mol" src="https://user-images.githubusercontent.com/39069546/116624197-5d91e700-a8fc-11eb-88ce-5e0969587d9f.png">
<img width="350" alt="scaffold" src="https://user-images.githubusercontent.com/39069546/116624250-7a2e1f00-a8fc-11eb-9c08-ec05b896296b.png">
 </p>

This would be the output if we didn't allow flipping fragments with >1 constrained atom (the suggestion from the other PR), or if we didn't check for constrained children for single atom fragments.
<img width="384" alt="Screen Shot 2021-04-21 at 3 30 18 PM" src="https://user-images.githubusercontent.com/39069546/116625317-49e78000-a8fe-11eb-92be-ac13dee81c4d.png">

And here is the output after these changes:
<img width="458" alt="desired" src="https://user-images.githubusercontent.com/39069546/116626509-3210fb80-a900-11eb-8d26-f95418537e54.png">

My question is, which `CoordgenFragmentFlipDOF` causes the flip of the 13-14 bond in the first result, and the 'flip' of atoms 4-11 in the second result?

Reply:
in the first case, if you mean why that bond is not pointing down, that's a flip around the 12-13 bond, i.e. the inversion of the fragment containing 13-14.
second case is a flip around  the 2-3 bond, so the inversion of the fragment containing atom 3.

Reply:
Ah, ok, thank you!